oxidation/libparsec/crates/crypto/src/rustcrypto/sequester.rs

```diff
@@ -44,15 +37,25 @@ impl TryFrom<&[u8]> for SequesterPrivateKeyDer {
 }
 
 impl EnforceDeserialize for SequesterPrivateKeyDer {
-    const SIZE_IN_BITS: usize = Self::SIZE_IN_BITS;
     const ALGORITHM: &'static [u8] = b"RSAES-OAEP-XSALSA20-POLY1305";
+
+    fn size_in_bytes(&self) -> usize {
+        self.size_in_bytes()
+    }
```

Comment:
Are you sure about that ?

Reply:
My concern here is that calling `size_in_bytes` would cause a recursion.

I would prefer to have the `size_in_bytes` explicitly called like:

```rust
impl EnforceDeserialize for SequesterPrivateKeyDer {
    fn size_in_bytes(&self) -> usize {
        self.0.n().bits() / 8
    }
}

impl SequesterPrivateKeyDer {
    pub fn size_in_bytes(&self) -> usize {
        <self as EnforceDeserialize>.size_in_bytes()
    }
   // ...
}
```

Reply:
Done